resources/sdk/webmessagingjava/templates/api_json.mustache

```diff
@@ -0,0 +1 @@
+/Users/jshaw/genesys_src/repos/purecloudlabs/platform-client-sdk-common/resources/templates/api_json.mustache
```

Comment:
Could you do it as a relative sym link?

Reply:
Ah I see what you mean.  This is my first exposure to symlinks so I'm fumbling a bit. Will do.